src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.h

```diff
@@ -151,6 +151,8 @@ class FileWatcherCertificateProvider final
 
   UniqueTypeName type() const override;
 
+  int64_t TestOnlyGetRefreshIntervalSecond() const;
```

Comment:
Most getters in the repo follow the snake style. Also this member function is not exposed as a public API since it's not in any include headers. Following the distributor() style above, do you think refresh_interval_seconds() would make more sense?

Reply:
Discussed style offline with other grpc folks, we'll keep it this way at their suggestion